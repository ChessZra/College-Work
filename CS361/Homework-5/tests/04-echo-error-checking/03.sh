#!/bin/bash

# body of the response for echo would be too large
# request should be 413

PORT=$@

EOL=$'\r\n'
REQUEST=$'GET /echo HTTP/1.1'${EOL}$'Header1: '`printf "%1070s" | tr " " "U"`${EOL}${EOL}

printf "$REQUEST" | nc -N 127.0.0.1 $PORT 2>/dev/null
