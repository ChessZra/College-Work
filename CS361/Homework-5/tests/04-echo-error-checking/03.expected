HTTP/1.1 413 Request Entity Too Large

